makefile

```diff
@@ -0,0 +1,90 @@
+.PHONY: up down build start stop logs ps prisma-generate prisma-migrate prisma-studio init-db shell-api rebuild clean
```

Comment:
Hope everybody like makefiles, I think it's much better to handle common commands